app/models/user.rb

```diff
@@ -37,4 +38,16 @@ def send_notification(change)
     ChangeMailer.page(change: change, user: self).deliver_later
   end
 
+  def email_domain_is_approved
+    user_domain = email.strip.split('@')[-1].downcase
```

Comment:
This will also allow me to sign up with the address `"themarshallproject.org"` if that is one of the approved domains. This is not a security problem since obviously that's not a valid email address, but it is kind of odd.